prov/util/src/util_mr.c

```diff
@@ -239,10 +246,7 @@ int ofi_mr_init(const struct fi_provider *in_prov, enum fi_mr_mode mode,
 
 void ofi_mr_close(struct ofi_util_mr *in_mr_h)
 {
-    if (!in_mr_h) {
-        FI_WARN(in_mr_h->prov, FI_LOG_MR, "util mr_close: received NULL input\n");
-        return;
-    }
+    assert(in_mr_h);
```

Comment:
ah, thanks for the pointer with core_prov. I wasn't sure what other option I had